client/services/bikes.tsx

```diff
@@ -12,6 +13,18 @@ const getBikes = async () => {
     }
 };
 
+const getBikesWithinProximity = async (radius, userLocation) => {
+    try {
+        const response = await axios.get('/bike');
```

Comment:
I will make another version of this so that you can just pass in bike location, user location and radius since for the List Bikes and the Bike Info page the user already has the bike data.